.gitignore

```diff
@@ -0,0 +1,5 @@
+object/*.o
```

Comment:
Found this useful when developing, but can remove it if preferred

Reply:
We just removed .gitignore from repo :)